src/auth/oauth2client.ts

```diff
@@ -1015,9 +1015,12 @@ export class OAuth2Client extends AuthClient {
    */
   async getTokenInfo(accessToken: string): Promise<TokenInfo> {
     const {data} = await this.transporter.request<TokenInfoRequest>({
-      method: 'GET',
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+      },
       url: OAuth2Client.GOOGLE_TOKEN_INFO_URL,
-      params: {access_token: accessToken},
+      data: querystring.stringify({access_token: accessToken}),
```

Comment:
You don't need to stringify here, `gaxios` does that for you :)  Just passing the object should suffice.

Reply:
It seems to stringify it as JSON based on the nock expectations:

```
"headers": {
    "content-type": [
      "application/json"
    ],
    "user-agent": [
      "google-api-nodejs-client/6.0.3"
    ],
    "x-goog-api-client": [
      "gl-node/10.15.3 auth/6.0.3"
    ],
    "accept": [
      "application/json"
    ],
    "content-length": [
      "22"
    ],
    "accept-encoding": [
      "gzip,deflate"
    ],
    "connection": [
      "close"
    ]
  },
  "body": "{\"access_token\":\"abc\"}"
}
```
But it does not form url-encode it to `"access_token=abc"`

A same technique seems to be used here to for form url-encoding:
https://github.com/googleapis/google-auth-library-nodejs/blob/b82db030935204c85ec7acc77542c21152aec074/src/auth/oauth2client.ts#L620